system-tests/project-fixtures/svelte/src/mount.cy.js

```diff
@@ -83,4 +83,25 @@ describe("Svelte mount", () => {
       );
     });
   });
+
+  context('teardown', () => {
+    beforeEach(() => {
+      // component-index.html has anchor element within [data-cy-root] so base # of elements is 1
+      cy.get('[data-cy-root]').children().should('have.length', 1)
+    })
+
+    it("should mount", () => {
+      cy.mount(Counter);
+    });
```

Comment:
```suggestion
    it('should mount', () => {
      cy.mount(Counter);
    });
```

Reply:
Addressed in [ce2c587](https://github.com/cypress-io/cypress/pull/24470/commits/ce2c5876323a89d9600bfc6bb21e539d7d9bded3)